### PR TITLE
Update DOM.js

### PR DESCRIPTION
**Pull request by strarsis**
This improves aliasing/reduces jagged edges when images are scaled.
see http://stackoverflow.com/questions/6492027/css-transform-jagged-edges-in-chrome

### CODE REVIEW

File: src/DOM.js [hunk 1]
```diff
@@ -53,7 +53,13 @@ export default class KenBurnsDOM extends KenBurnsBase<Source> {
       this.container.innerHTML = "";
       this.container.appendChild(source);
       source.style.position = "absolute";
+
       source.style.willChange = "transform"; // already supported by some browsers, ensures GPU acceleration
+      source.style.outline = "1px solid transparent";         // improves edges
+      source.style.backfaceVisibility = 'hidden';             // "
+      source.style['-webkit-backface-visibility'] = 'hidden'; // "
+      source.style.boxShadow = '0 0 1px rgba(255,255,255,0)'; // "
```
Comment (gre) [inline src/DOM.js]: minor code style note: can you switch ' to "  and remove the `// "` ? a link to the stackoverflow in the comment can be good to track this for later too :)

**Comment by gre** [comment]
thanks for this !

**Review by gre — CHANGES_REQUESTED** [review]

File: src/DOM.js (hunk 1 above)
Comment (gre) [inline src/DOM.js]: Also, I don't think `.style['-webkit-backface-visibility']` works. I think it should be `.style.WebkitBackfaceVisibility`

Comment (gre) [inline src/DOM.js]: but it does not seem to exist anymore, not in chrome at least, is if for another browser?

Comment (gre) [inline src/DOM.js]: yeah, it exists in Safari

screen shot 2016-12-22 at 12 41 30

**Comment by strarsis** [comment]
Yes, I removed the // " and formatted those lines into one block with one comment and link to the SO thread, quotes were changed to double-quotes.

The css property access is a bit confusing, it appears that the property name has to start with an uppercase letter in order to prefix with '-' (http://stackoverflow.com/questions/9438949/access-the-webkit-vendor-prefix-in-javascript).

**Comment by gre** [comment]
interesting, I didn't know about this uppercase thing.

**Pull request merged**